spark/dl/src/main/scala/com/intel/analytics/bigdl/transform/vision/image/label/roi/RoiLabel.scala

```diff
@@ -28,33 +28,35 @@ import com.intel.analytics.bigdl.utils.{T, Table}
  * @param masks the array of annotation masks of the targets
  */
 case class RoiLabel(classes: Tensor[Float], bboxes: Tensor[Float],
-  masks: Option[Array[Tensor[Byte]]] = None) {
+  masks: Array[Tensor[Byte]] = null) {
   def copy(target: RoiLabel): Unit = {
     classes.resizeAs(target.classes).copy(target.classes)
     bboxes.resizeAs(target.bboxes).copy(target.bboxes)
-    require(!target.masks.isDefined, "Do not support support copying RoiLabels with masks")
+    require(target.masks == null, "Do not support support copying RoiLabels with masks")
```

Comment:
=> "RoiLabels with masks not supported" ?

Reply:
Yes, the mask array is `val`, so we cannot change the size of the array. What if the user wants to copy the mask with a different size?

Reply:
No,  I meant to update the log to above one